File: applicatie/src/main/java/com/vdzon/administratie/pdfgenerator/TabelCols.java
package com.vdzon.administratie.pdfgenerator;

public class TabelCols {
    private int[] col = new int[10];

    public TabelCols(int col1, int col2, int col3) {
        col[0] = col1;
        col[1] = col2;
        col[2] = col3;
    }

    public TabelCols(int col1, int col2, int col3, int col4, int col5) {
        col[0] = col1;
        col[1] = col2;
        col[2] = col3;
        col[3] = col4;
        col[4] = col5;
    }

    public int getCol(int i){
        return col[i];
    }

}

File: pdfgenerator/impl/src/main/java/com/vdzon/administratie/pdfgenerator/pdfutil/TabelCols.kt
package com.vdzon.administratie.pdfgenerator.pdfutil

class TabelCols {
    private val col = IntArray(10)

    constructor(col1: Int, col2: Int, col3: Int) {
        col[0] = col1
        col[1] = col2
        col[2] = col3
    }

    constructor(col1: Int, col2: Int, col3: Int, col4: Int, col5: Int) {
        col[0] = col1
        col[1] = col2
        col[2] = col3
        col[3] = col4
        col[4] = col5
    }

    fun getCol(i: Int): Int {
        return col[i]
    }

}
